pkg/networkmanager/dialogs-common.jsx

```diff
@@ -234,6 +236,8 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
             dlg = <BridgeDialog {...properties} />;
         else if (type == 'wg')
             dlg = <WireGuardDialog {...properties} />;
+        else if (type == 'openvpn')
+            dlg = <OpenVPNDialog {...properties} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19363-20230921-075022-79cb9fd2-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)